README.md

```diff
@@ -1,10 +1,10 @@
 ## Orchestrator Helm Repository
 Helm chart to deploy the Orchestrator solution suite. The following components will be installed on the cluster:
-- Janus IDP backstage
-- SonataFlow Operator (with Data-Index and Job Service)
+- RHDH (Red Hat Developer Hub) Backstage
```

Comment:
Let's keep it only for stable branch

Reply:
Why? This chart is dedicated to OCP and uses operators from redhat-operators marketplace. This is a correct statement.
I can add another bullet for now for "plugins from janus-idp NPM registry"